src/main/resources/fabric.mod.json

```diff
@@ -1,20 +1,20 @@
 {
   "schemaVersion": 1,
-  "id": "modid",
+  "id": "emcutils",
   "version": "${version}",
 
   "name": "Empire Minecraft Utilities",
-  "description": "This is an example description! Tell everyone what your mod is about!",
+  "description": "A collection of various utilities for the Empire Minecraft server.",
   "authors": [
-    "Me!"
```

Comment:
But... it is me :(

Reply:
Yeah, but... 
![do you really want this?](https://i.imgur.com/rLWUs8n.png)

Reply:
I admit defeat